4.26/platform_isv.html

```diff
@@ -168,6 +168,37 @@ <h2 id="possible-deadlock-in-ijobchangelistener">Possible deadlock in IJobChange
     </td>
   </tr>
   <!-- *********************** End of SWT *********************** -->
+
+  <!-- ******************* General Updates ************************************* -->
+  <tr>
+    <td id="GeneralUpdates" class="section" colspan="2">
```

Comment:
Please add <li><a href="#GeneralUpdates">General Updates</a></li> at the top (line 28) to link to this new section.

Reply:
done via https://github.com/eclipse-platform/www.eclipse.org-eclipse-news/pull/72

Reply:
Thank you!

Actually I validated the html and that wasn't an issue. :/